test/integration/__tests__/live-api.js

```diff
@@ -618,9 +621,21 @@ describe('API integration', () => {
         const newPath = '/newPath'
         const newUrl = `${server.url}${newPath}`
 
+        // I think there are some eventual consistency issues here
+        await new Promise((resolve) => setTimeout(resolve, 1000))
+
+        const result = await client.getAssembly(assemblyId)
+        console.log(result)
+
+        expect(result.notify_status).toBe('successful')
+        expect(result.notify_response_code).toBe(200)
+
         if (secondNotification) {
           expect(path).toBe(newPath)
 
+          // For some reason, notify_url doesn't get updated to new URL
+          // expect(result.notify_url).toBe(newUrl)
```

Comment:
had to comment out this line

Reply:
Yeah, we have that notification API now where you can request further notifications and their URLs. The original notify_url is no longer changed.

Reply:
Because the other notify_start, etc. fields also do not get changed. It's not a bug, it's a feature. :)

Reply:
Ok, i'll change the comment